### ISSUE TITLE
Keep support for build_image in shippable.yml

### ISSUE THREAD

**Issue by mbrodala** [Shippable/support#2049]
In [this blog post](http://blog.shippable.com/shippable-3.0-breaking-changes#BuildImage) various tags within `shippable.yml` where deprecated, among others also `build_image`.

I would like to request to not deprecate this tag anymore and keep supporting it. Reason is this: within the `shippable.yml` i configure the build environment, e.g. what language to use and what version as well as manually pulling in dependencies for the build. From my POV the image to use heavily affects the build and should thus be kept together with all other settings.

Having this image in the project settings only has the disadvantage that one can easily miss configuring it (even though one e.g. uses always the same image) and must remember to manually update it in case the build must be changed in some way. An example for the latter may be major bumping the version of a language and switching the image for support during builds.

In our case we work exclusively with PHP projects and for *every* enabled project we must remember to set the `shippableimages/ubuntu1404_php` image in the project settings. It would help if one could change the default image for enabled projects but this wouldn't cover the other usecase mentioned above.

Please consider this for a better user experience with Shippable.

**Comment by manishas** [comment]
@mbrodala you are right, we intended removing support for build_image from the yml and move it to the project settings UI, so we wrote a blog about it.

However, based on feedback, we have decided to continue supporting the build_image tag. In fact, we're also going to move the other docker settings to the yml and if a customer configures UI and yml, yml will win. This is coming up in the next few weeks.
